neutrino.go

```diff
@@ -86,6 +86,22 @@ var (
 	DefaultBlockCacheSize uint64 = 4096 * 10 * 1000 // 40 MB
 )
 
+// SideLoadOpt defines the config required to sideload headers into the DB.
+type SideLoadOpt struct {
+	// SkipVerify indicates if we are to run verification on the headers from the
+	// sideLoad source.
+	SkipVerify bool
+
+	// Path contains the location of the sideload resource.
+	Path string
+
+	// Enabled indicates if sideloading is turned on or off
+	Enabled bool
```

Comment:
If we make the struct a pointer, then we can use the existence of the pointer to determine if this was enabled or not.